README.md

```diff
@@ -115,7 +117,43 @@ diff-table \
   -events
 ```
 
-The output is a newline-delimited set of JSON-encoded events. Column-based changes will always come first. The `type` field denotes the type of event which can be switched on during consumption.
+The output is a newline-delimited set of JSON-encoded events. Column-based changes will always come first. The `type` field denotes the type of event which can be switched on during consumption. The base event structured looks as follows:
+
+```js
+{
+  // Event name/type.
+  "type": "row-added",
+
+  // Unix epoch timestamp in seconds.
+  "time": 1520114848,
+
+  // Offset of the row in the batch, if applicable. This can also be used as an
+  // ordering mechanism.
+  "offset": 1,
+
+  // Key of the row this event pertains to, if applicable.
+  "key": {...},
+
+  // Full snapshot of the row. This will be present row row-added events, but
+  // can be optionally included for row-changed and row-removed events.
```

Comment:
> can be optionally included for row-changed and row-removed events

Is this option already implemented or did I miss it in these changes?

Reply:
Good catch! Adding..

Reply:
Added a new option to support this.